composer.json

```diff
@@ -29,19 +29,22 @@
     },
     "require": {
         "php": "^7.3 || ~8.0.0",
+        "ext-json": "*",
```

Comment:
Why is this added? Is it because of the body params `JsonStrategy`? If so, that's considered opt-in; not everyone will use it. I'd say this should instead become:

- A require-dev requirement AND
- A suggest entry